### ISSUE TITLE
Using PERCY_ENABLE can lead to NPE

### ISSUE THREAD

**Issue by SemaLuna** [percy/cli#1560]
## The problem

When using `PERCY_ENABLE` to disable Percy, the `exec` command unfortunately still has logic for trying to send buildEvents upstream (when there is an error).

So if the command that percy wraps on errors, we get a NPE.

Eventually leading to an NPE:
```
file:///PATH_TO_PROJECT/node_modules/.pnpm/@percy+cli-exec@1.28.0_typescript@5.4.2/node_modules/@percy/cli-exec/dist/exec.js:145
          percy.client.sendBuildEvents(percy.build.id, myObject);
                ^
TypeError: Cannot read properties of undefined (reading 'client')
    at ChildProcess.<anonymous> (file:///PATH_TO_PROJECT/node_modules/.pnpm/@percy+cli-exec@1.28.0_typescript@5.4.2/node_modules/@percy/cli-exec/dist/exec.js:145:17)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1100:16)
    at Socket.<anonymous> (node:internal/child_process:458:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:301:12)
```

## Environment

- `@percy/cli` version: 1.28.0

## Code to reproduce issue

It is easy enough to notice the issue in the code - in [this line](https://github.com/percy/cli/blob/master/packages/cli-exec/src/exec.js#L145) we try to sendBuildEvents, however `percy` is undefined when `PERCY_ENABLE` is 0 (you can see this assumption in the same file [here](https://github.com/percy/cli/blob/master/packages/cli-exec/src/exec.js#L62)) 

Should be a simple fix.

**Comment by prklm10** [comment]
Hi @SemaLuna, Sorry for the late revert we will fix look into this and the fix will be available in the next stable release.

**Comment by SemaLuna** [comment]
@prklm10 no problem at all - thanks for the fix.

**Comment by prklm10** [comment]
Hi @SemaLuna we have fixed this bug in 1.28.3.

**Comment by prklm10** [comment]
Closing this issue.
